### DISCUSSION TITLE
为什么很多人的配置DNS: fallback-filter ipcidr: 写了诸如240.0.0.0/4的特殊网段啊

### DISCUSSION THREAD

**Discussion by MJRT** [MetaCubeX/mihomo#1227] [Q&A]
我看[wiki](https://wiki.metacubex.one/config/dns/?h=fallback)的例子也有写的, 还有网上分享的配置(比如:[eg1](https://github.com/MetaCubeX/mihomo/discussions/1189) )里这一项有更多的[保留IP地址](https://zh.wikipedia.org/wiki/%E4%BF%9D%E7%95%99IP%E5%9C%B0%E5%9D%80)

为什么这些IP被认为已经被污染呢, 不是应该直接走LAN/DIRECT么?

**Comment by pillarcoin** [comment]
根据 [ipv4的Wikipedia](https://en.wikipedia.org/wiki/Internet_Protocol_version_4#:~:text=14%5D%5B15%5D-,240.0.0.0/4,-240.0.0.0%E2%80%93255.255.255.254)  
`240.0.0.0/4`这个网段属于保留地址，也就是老的E类网络，虽然现在不这么分类。所以如果你dns解析的ip在这个网段内的话，就可以被认为是被污染的，因为没有人会用这个ip地址
